tools/project_manager/post_files.inc

```diff
@@ -107,13 +107,14 @@ function generate_interim_file($projectid, $limit_round_id, $which_text, $includ
     fclose($handle);
     // make the zip and sent it to the user
     header("Content-type: application/octet-stream");   
-    header("Content-disposition: attachment; filename={$filename}.zip");   
-    passthru("zip -q -j - $textfile_path", $return_code);
-    if ($return_code != 0)
-    {
-        echo "post_files: the zip command failed.";
-    }
-    // cleanup will get called whatever happens  
+    header("Content-disposition: attachment; filename={$filename}.zip");
+
+    $zip_path = "{$dirname}/{$filename}.zip";
+    create_zip_from([$textfile_path], $zip_path);
```

Comment:
This is the contents of the generated zip file:

![image](https://user-images.githubusercontent.com/3049847/62069159-eff0ec80-b237-11e9-975c-289f0952a119.png)

Would it be better to fix this somehow in the `create_zip_from` function or by using [`chdir`](https://www.php.net/manual/en/function.chdir.php) to change the directory to `./tmp/PROJECT_DIR` before creating the zip archive?

Reply:
This is most likely problematic in a couple places, so it is probably a good idea to think it through.

Reply:
I do think we want the ability to create a directory-less zip archive (ie: just the file(s) in question in the zip, not the directory). I think it makes sense to put this functionality within `create_zip_from()`.

Reply:
Turns out that was easier than I thought it was going to be. Fixed in 6ae7e0a8a6467f49381d11f7512e91f6e26b9461. Now all files added to the zip archive *will* go into the root directory, is that okay? 

Anything more complicated is probably best done directly through `ZipArchive` or through a more specific method. 

![image](https://user-images.githubusercontent.com/3049847/62155340-e1bed100-b308-11e9-8d54-22a0cf40040b.png)

Reply:
I think that's fine. That's certainly how we are using `zip` currently by the looks of it.